deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -192,6 +192,19 @@ config:
     name:
     key:
 
+  # -- Configuration section for the Venafi Kubernetes Agent itself
+  venafiConnection:
+    # -- When set to false, the rendered output does not contain the
+    # VenafiConnection CRDs and RBAC. This is useful for when the
+    # Venafi Connection resources are already installed separately.
+    enabled: false
```

Comment:
This description is out of date. 
`enabled` doesn't match Michaels documentation which uses `enable`.
I prefer `enabled` because it is consistent with the `crds.enabled` flag in cert-manager.